src/platforms-plugins.json

```diff
@@ -902,7 +903,7 @@
             "I2C": "yes",
             "One Wire": "no",
             "Stepper": "no",
-            "Serial/UART": "no",
+            "Serial/UART": "no*",
```

Comment:
Remove the \*, these are used as keys to apply a color class to the generated table cells